src/org/zaproxy/zap/extension/bugtracker/resources/help/contents/bugtracker.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+<TITLE>Bug Tracker</TITLE>
+</HEAD>
+<BODY BGCOLOR="#ffffff">
+<H1>Bug Tracker</H1>
+<p>
+The Bug Tracker add-on can help the user raise the Alerts that they recieve in the form of an issue or a bug on a Bug Tracker (like Bugzilla, Github,etc) while scanning a web-app.</p><br>
+
+<H2>Configuration of Bug Trackers</H2>
+<p>
+The user may configure the Bug Trackers with their important details like Username, Password, Repository URL, etc by going to <b>Options->Bug Tracker</b> and then selecting the appropriate Bug Tracker.
+</p><br>
+
+<H2>Raising the Issues</H2>
+<p>
+The user can select one or more Alerts (even their subnodes as per need) after scanning then <b>Right Click -> Raise Issue on a Bug Tracker.</b><br>
```

Comment:
Could say "context menu item" instead of "right click" (the context menu can be invoked with keyboard too).